setup.py

```diff
@@ -16,19 +16,19 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
-    'check-manifest>=0.25',
-    'coverage>=4.0',
-    'isort>=4.2.2',
     'mock>=1.3.0',
-    'pydocstyle>=1.0.0',
-    'pytest-cov>=1.8.0',
-    'pytest-pep8>=1.0.6',
-    'pytest>=3.8.0,<5.0.0',
+    'pytest-invenio>=1.3.4',
+    # TODO: Remove all lines below with pytest-invenio v1.4.0:
+    "pytest-cov>=2.10.1",
+    "pytest-isort>=1.2.0",
+    "pytest-pycodestyle>=2.2.0",
+    "pytest-pydocstyle>=2.2.0",
+    "pytest>=6,<7",
 ]
 
 extras_require = {
     'docs': [
-        "Sphinx>=1.4.2",
+        "Sphinx>=3",
```

Comment:
Is this something we should start to integrate systematically? many modules have `< 3`

Reply:
yes